### PR TITLE
Recognize and correctly ignore x services attributes 1425959804

### PR DESCRIPTION
**Pull request by mastfish**

### CODE REVIEW

**Comment by gingermusketeer** [comment]
Is the consistency check going to be changed to take into account the x-services values? e.g ignore inconsistencies with the service being changed

**Comment by mastfish** [comment]
Yes - that will come soon.

File: lib/apivore/rspec_matchers.rb [hunk 1]
```diff
@@ -41,7 +41,10 @@ module RspecMatchers
 
       match do |body|
         our_swagger = JSON.parse(body)
-        master_definitions = master_swagger["definitions"]
+        master_definitions = master_swagger["definitions"].transform_values do |definition|
+          # 'x-services' is added by api.westfield.io - services shouldn't need to define it
+          definition.tap{|d|d.delete 'x-services'}
+        end
```
Comment (gingermusketeer) [inline lib/apivore/rspec_matchers.rb]: Perhaps this should be extracted into a separate function `strip_x_services`

**Comment by mastfish** [comment]
This is somewhat half baked - closing while I work on it.

**Pull request closed without merging**